components/openzwave-adapter/src/lib.rs

```diff
@@ -304,9 +312,12 @@ impl OpenzwaveAdapter {
                         include_map.push(include_setter_id.clone(), controller);
 
                         box_manager.add_channel(Channel {
-                            kind: ChannelKind::ZwaveInclude,
-                            supports_send: true,
-                            ..Channel::empty(&include_setter_id, &service_id, &adapter_id)
+                            implements: TaxoId::new("zwave/include"),
```

Comment:
Anything non-reserved (eg. `kind`, `type` or `feature`) will work well for client side, but don't bother if there is too much to rename :)

Maybe it's just me, but `channel's feature` makes me think for a moment that channel can have several `features` :)